package.json

```diff
@@ -10,7 +10,7 @@
     "test-cov-html": "lab -r html -o coverage.html"
   },
   "engines": {
-    "node": ">= 6.0.0 <13.0.0"
+    "node": ">10.8.0 <15.0.0"
```

Comment:
If we adding here node 14, maybe is it good to add it to the build as well?

Reply:
oh, actually we can’t support node 14 yet because our node-sass fork is not built for node 14 yet, I'll update this line to '>10.8.0 <13.0.0'